scripts/mujoco/requirements.txt

```diff
@@ -0,0 +1,138 @@
+absl-py==2.1.0
+ale-py==0.10.1
+annoy==1.17.3
+blitzdb==0.4.4
+bottle==0.13.2
+box2d-py==2.3.5
+cachetools==5.5.0
+certifi==2024.8.30
+cffi==1.17.1
+charset-normalizer==3.4.0
+chex==0.1.87
+click==8.1.7
+cloudpickle==3.1.0
+colorlover==0.3.0
+contourpy==1.3.0
+cycler==0.12.1
+Cython==0.29.37
+decorator==4.4.2
+docker-pycreds==0.4.0
+etils==1.9.4
+Farama-Notifications==0.0.4
+fasteners==0.19
+filelock==3.16.1
+flax==0.9.0
+fonttools==4.54.1
+fsspec==2024.9.0
+gitdb==4.0.11
+GitPython==3.1.43
+glfw==2.7.0
+google-api-core==2.21.0
+google-auth==2.35.0
+google-cloud-core==2.4.1
+google-cloud-storage==2.18.2
+google-crc32c==1.6.0
+google-resumable-media==2.7.2
+googleapis-common-protos==1.65.0
+grpcio==1.66.2
+gymnasium==1.0.0
+h5py==3.12.1
+huggingface-hub==0.25.2
+huggingface-sb3==3.0
+humanize==4.11.0
+idna==3.10
+imageio==2.35.1
+imageio-ffmpeg==0.5.1
+importlib_resources==6.4.5
+jax==0.4.34
+jaxlib==0.4.34
+Jinja2==3.1.4
+joblib==1.4.2
+kiwisolver==1.4.7
+lazy_loader==0.4
+lime==0.2.0.1
+Markdown==3.7
+markdown-it-py==3.0.0
+MarkupSafe==3.0.1
+matplotlib==3.9.2
+mdurl==0.1.2
+minari==0.5.1
+ml_dtypes==0.5.0
+moviepy==1.0.3
+mpmath==1.3.0
+msgpack==1.1.0
+mujoco==3.2.3
+mujoco-py==2.1.2.14
+nest-asyncio==1.6.0
+networkx==3.4.1
+numpy==1.26.4
+nvidia-cublas-cu12==12.1.3.1
+nvidia-cuda-cupti-cu12==12.1.105
+nvidia-cuda-nvrtc-cu12==12.1.105
+nvidia-cuda-runtime-cu12==12.1.105
+nvidia-cudnn-cu12==9.1.0.70
+nvidia-cufft-cu12==11.0.2.54
+nvidia-curand-cu12==10.3.2.106
+nvidia-cusolver-cu12==11.4.5.107
+nvidia-cusparse-cu12==12.1.0.106
+nvidia-nccl-cu12==2.20.5
+nvidia-nvjitlink-cu12==12.6.77
+nvidia-nvtx-cu12==12.1.105
+opencv-python==4.10.0.84
+opt_einsum==3.4.0
+optax==0.2.3
+orbax-checkpoint==0.7.0
+packaging==24.1
+pandas==2.2.3
+pillow==10.4.0
+platformdirs==4.3.6
+plotly==5.24.1
+proglog==0.1.10
+proto-plus==1.24.0
+protobuf==5.28.2
+psutil==6.0.0
+pyarrow==17.0.0
+pyasn1==0.6.1
+pyasn1_modules==0.4.1
+pycparser==2.22
+pygame==2.6.1
+Pygments==2.18.0
+PyOpenGL==3.1.7
+pyparsing==3.1.4
+python-dateutil==2.9.0.post0
+pytz==2024.2
+PyYAML==6.0.2
+reach==4.1.1
+requests==2.32.3
+rich==13.9.2
+rsa==4.9
+scikit-image==0.24.0
+scikit-learn==1.5.2
+scipy==1.14.1
+sentry-sdk==2.16.0
+setproctitle==1.3.3
+setuptools==75.1.0
+shellingham==1.5.4
+six==1.16.0
+smmap==5.0.1
+stable_baselines3@git+https://github.com/pseudo-rnd-thoughts/stable-baselines3.git@2f6246033cb9111ad0d0df9d9bd98555957e08b5#egg=stable_baselines3
+swig==4.2.1
+sympy==1.13.3
+tenacity==9.0.0
+tensorboard==2.18.0
+tensorboard-data-server==0.7.2
+tensorstore==0.1.66
+threadpoolctl==3.5.0
+tifffile==2024.9.20
+toolz==1.0.0
+torch==2.4.1
+tqdm==4.66.5
+triton==3.0.0
+typer==0.12.5
+typing_extensions==4.12.2
+tzdata==2024.2
+urllib3==2.2.3
+wandb==0.18.3
+wasabi==1.1.3
+Werkzeug==3.0.4
+zipp==3.20.2
```

Comment:
many deps are not used.

I would rather put here only the libraries that we are importing in the other file. Otherwise, for simplicity, you can just remove this

Reply:
I have been burned before for not including a `requirments.txt`

There is no downside to include, you are not forced to use it.